VideoSampleCaptureRender/ViewController.swift

```diff
@@ -278,13 +282,23 @@ class ViewController: UIViewController, UITextFieldDelegate {
 extension ViewController: TwilioConversationsClientDelegate {
     func conversationsClient(conversationsClient: TwilioConversationsClient,
         didFailToStartListeningWithError error: NSError) {
+        
+        // Do not interrupt the on going conversation UI. Client status will
+        // changed to .FailedToListen when conversation ends.
+        if (conversation == nil) {
             self.updateClientStatus(.FailedToListen, animated: false)
+        }
     }
     
     func conversationsClientDidStartListeningForInvites(conversationsClient: TwilioConversationsClient) {
         // Successfully listening for Invites
         self.setupLocalMedia()
-        self.updateClientStatus(.Listening, animated: true)
+        
+        // Do not interrupt the on going conversation UI. Client status will
+        // changed to .Listening when conversation ends.
+        if (conversation == nil) {
+            self.updateClientStatus(.Listening, animated: true)
+        }
```

Comment:
You shouldn't setup your local media in this call because the Client might lose, and regain connectivity several times.